src/hearings/components/multi-level-selector/multi-level-selector.component.spec.ts

```diff
@@ -0,0 +1,216 @@
+      },
+      {
+        key: 'GeneralPractitioner',
+        value_en: 'General Practitioner',
+        value_cy: '',
+        hintText_EN: 'true',
+        hintTextCY: 'false',
+        order: 1,
+        parentKey: '3',
+        child_nodes: [],
+        selected: false,
+      }]
+    }, {
+      key: 'MedicallyQualifiedPanelMember',
+      value_en: 'Medically qualified panel member',
+      value_cy: '',
+      hintText_EN: 'true',
+      hintTextCY: 'false',
+      order: 1,
+      parentKey: '3',
+      selected: false,
+      child_nodes: [{
+        key: 'Cardiologist',
+        value_en: 'Cardiologist',
+        value_cy: '',
+        hintText_EN: 'true',
+        hintTextCY: 'false',
+        order: 1,
+        parentKey: '3',
+        child_nodes: [],
+        selected: false,
+      },
+      {
+        key: 'Carer',
+        value_en: 'Carer',
+        value_cy: '',
+        hintText_EN: 'true',
+        hintTextCY: 'false',
+        order: 1,
+        parentKey: '3',
+        child_nodes: [],
+        selected: false,
+      },
+      {
+        key: 'EyeSurgeon',
+        value_en: 'Medically qualified panel member',
+        value_cy: '',
+        hintText_EN: 'true',
+        hintTextCY: 'false',
+        order: 1,
+        parentKey: '3',
+        child_nodes: [],
+        selected: false,
+      },
+      {
+        key: 'GeneralPractitioner',
+        value_en: 'General Practitioner',
+        value_cy: '',
+        hintText_EN: 'true',
+        hintTextCY: 'false',
+        order: 1,
+        parentKey: '3',
+        child_nodes: [],
+        selected: false,
+      }]
+    }, {
+      key: 'FinanciallyQualifiedPanelMember',
+      value_en: 'Financially qualified panel member',
+      value_cy: '',
+      hintText_EN: 'true',
+      hintTextCY: 'false',
+      order: 1,
+      parentKey: '3',
+      child_nodes: [],
+      selected: false,
+    },
+    {
+      key: 'RegionalMedicalMember',
+      value_en: 'Regional Medical Member',
+      value_cy: '',
+      hintText_EN: 'true',
+      hintTextCY: 'false',
+      order: 1,
+      parentKey: '3',
+      child_nodes: [],
+      selected: false,
+    }];
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [ReactiveFormsModule],
+      declarations: [MultiLevelSelectorComponent],
+      providers: [
+        FormBuilder
+      ]
+    })
+      .compileComponents();
+    fixture = TestBed.createComponent(MultiLevelSelectorComponent);
+    component = fixture.componentInstance;
+    const modelConvertor = new DataModelConvertor(component.fb);
+    component.level = 1;
+    component.multiSelect = modelConvertor.convertRefDataModelToArray(LIST_OFF_VALUES_REF);
+    component.configLevels = [
+      {
+        controlType: ControlTypeEnum.CHECK_BOX,
+        level: 1,
+      },
+      {
+        controlType: ControlTypeEnum.SELECT,
+        level: 2,
+      }
+    ];
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Are we actually testing the component 😭.

Reply:
Unit test result is ok and Code coverage is 95%. So yes. Are you expecting more tests?

Reply:
Kinder maybe we can test elements rendered, given inputs provided.